src/components/shared/Hero.js

```diff
@@ -0,0 +1,38 @@
+import { useResponsive } from 'hooks/useResponsive'
+import { Box } from 'grommet'
+
+export const Hero = ({
+  header,
+  body,
+  boxPadding = {},
+  boxWidth = '100%',
```

Comment:
The parent (the outer div) takes `...props` and its width can be set by passing the Grommet `width` prop (or any other props that Grommet supports by default) . The `boxWidth` is specifically for setting the inner div's width (the white box). So it needs to stay otherwise, if we pass the Grommet's `width` prop to the Hero component, it will apply to both the inner div and the outer div. Let's keep the `boxWidth` for now, thank you!

e.g.,) `<Hero width='forOuterDiv' boxWdith="forInnerDv />`